coralproject/talk
# Bugfix/userdrawer changestatus

## What does this PR do?
the mutations for changing user status (ban/suspension) weren't requesting the history of bans/suspensions in response, which was fine for the community table view, but caused the empty popover behaviour we were seeing in the user drawer. 

## How do I test this PR?
- click on a user in the community table
- ban or suspend the user
- click on the "i" to see details of suspension or ban
